shell/shared/admin.js

```diff
@@ -313,6 +317,14 @@ if (Meteor.isClient) {
       Meteor.call("setAccountSetting", token, "github", event.target.githubLogin.checked, handleErrorBound);
       Meteor.call("setAccountSetting", token, "emailToken", event.target.emailTokenLogin.checked, handleErrorBound);
       Meteor.call("setSetting", token, "smtpUrl", event.target.smtpUrl.value, handleErrorBound);
+      if (globalDb.isFeatureKeyValid()) {
+        Meteor.call("setAccountSetting", token, "ldap", event.target.ldapLogin.checked, handleErrorBound);
+        Meteor.call("setLdapSetting", token, "Url", event.target.ldapUrl.value, handleErrorBound);
```

Comment:
Is there a good reason why these are split like this (half `setLdapSetting`, half `setSetting` with a setting name of `ldap*`)?  Seems surprising.

I'd maybe rather see this live in the body of `setSetting`, where you check the setting name against a list of settings that you reexport onto `LDAP_DEFAULTS`.  That way, you can't accidentally write client code that changes the DB but not the LDAP_DEFAULTS.